changelogs/fragments/56880-dnsmadeeasy-fix_monitor_handling.yml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - correctly handle monitor (DNS failover) record property
```

```suggestion
  - "dnsmadeeasy - correctly handle monitor (DNS failover) record property."
```
If you don't include the name, it will be totally unclear in the changelog what this entry is about.